src/include/fields_item.hxx

```diff
@@ -16,7 +16,7 @@
 
 enum
 {
-  POFI_BY_KIND = 2, // this item needs to be replicated by kind
+  POFI_BY_KIND = 3, // this item needs to be replicated by kind
```

Comment:
This is actually just an arbitrary flag (there used to be more than one), so this doesn't need to change (and hence shouldn't).